vassal-app/src/main/java/VASSAL/build/module/Map.java

```diff
@@ -3768,25 +3793,41 @@ public String getIdentifier() {
   }
 
 
+  /**
+   * Common view setup code for Map and PrivateMap, which confusingly have different View classes that sharing the same name
+   */
+  public void setUpView() {
+    scroll = new AdjustableSpeedScrollPane(
+      theMap,
+      JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED,
+      JScrollPane.HORIZONTAL_SCROLLBAR_AS_NEEDED);
+    scroll.unregisterKeyboardAction(
+      KeyStroke.getKeyStroke(KeyEvent.VK_PAGE_DOWN, 0));
+    scroll.unregisterKeyboardAction(
+      KeyStroke.getKeyStroke(KeyEvent.VK_PAGE_UP, 0));
+    scroll.setAlignmentX(0.0f);
+    scroll.setAlignmentY(0.0f);
+
+    layeredPane.setLayout(new InsetLayout(layeredPane, scroll));
+    layeredPane.add(scroll, JLayeredPane.DEFAULT_LAYER);
+
+    // When our Viewport changes for any reason, check if we need to update our "preferred center point"
+    scroll.getViewport().addChangeListener(new ChangeListener() {
```

Comment:
Change listeners can be replaced with lambdas:
```
scroll.getViewport().addChangeListener(e -> updateCenter());
```

Reply:
fixed